package.json

```diff
@@ -21,43 +20,37 @@
     "integ:centerpoint": "python3 lib/osml-model-runner-test/bin/process_image.py --image small --model centerpoint",
     "integ:flood": "python3 lib/osml-model-runner-test/bin/process_image.py --image large --model flood",
     "integ:meta": "python3 lib/osml-model-runner-test/bin/process_image.py --image meta --model centerpoint",
-    "integ:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --payload '{\"test_type\": \"integ\", \"source_image_key\": \"small.tif\"}' --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
-    "load:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --payload '{\"test_type\": \"load\", \"locust_image_keys\": \"small.tif,meta.ntf,sicd-capella-chip.ntf,sicd-interferometric-hh.nitf\", \"locust_headless\": \"true\", \"locust_run_time\": \"5m\", \"locust_users\": \"1\", \"locust_spawn_rate\": \"1\"}' --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
-    "mr:docker-build": "docker build lib/osml-model-runner/. -t osml-model-runner:local",
-    "mr:docker-run": "bash lib/osml-model-runner/scripts/run_container.sh",
-    "mr:monitor": "bash scripts/dev_monitor.sh",
-    "mr:globe": "npm run --prefix lib/osml-cesium-globe/ dev"
+    "integ:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --payload '{\"image_uri\": \"s3://osml-test-images-409719124294/small.tif\"}' --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di"
```

Comment:
now that this is ising the image uri instead of the image key, we should update this to use an ENV VAR for the image_uri instead of hard coding.  Also, I believe we need to add the cli-binary-format flag.  Something like
`    "integ:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --cli-binary-format raw-in-base64-out --payload \"{\\\"image_uri\\\": \\\"$TS_INTEG_IMAGE_URI\\\"}\" --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
`